src/silicone/time_projectors/__init__.py

```diff
@@ -7,3 +7,4 @@
 """
 
 from .extend_latest_time_quantile import ExtendLatestTimeQuantile  # noqa: F401
+from .extend_rms_closest import ExtendRMSClosest
```

Comment:
Our linter doesn't seem to mind, but adding the # noqa: F401 command means that using a proper linter won't complain that we're not going to get error messages about the fact that these imports aren't used here.